docs/installation-vagrant.rst

```diff
@@ -0,0 +1,119 @@
+
+Install and run everything
+--------------------------
+
+#. Install VirtualBox 4.x from http://www.virtualbox.org/
+
+   .. note:: (Windows) After installing VirtualBox you need to set
+              PATH=C:\\Program Files\\Oracle\\VirtualBox\\VBoxManage.exe;
+
+#. Install vagrant >= 1.6 using the installer from `vagrantup.com <http://vagrantup.com/>`_
+
+#. Clone Kitsune::
+
+       git clone https://www.github.com/mozilla/kitsune.git
+       cd kitsune
+
+#. Start the VM and install everything. (approx. 30 min on a fast net connection).::
+
+      vagrant up
+
+   .. note:: VirtualBox creates VMs in your system drive.
+             If it won't fit on your system drive, you will need to `change that directory to another drive <http://emptysquare.net/blog/moving-virtualbox-and-vagrant-to-an-external-drive/>`_.
+
+   At the end, you should see::
+
+      => default: notice: Finished catalog run in .... seconds
+
+
+   If the above process fails with an error, check `Errors`_.
+
+#. Log into the VM with ssh::
+
+       vagrant ssh
+
+#. Create a Superuser::
+
+       cd kitsune
+       source venv/bin/activate
+       ./manage.py createsuperuser
+   You should give the Username and Password for login into your local Kitsune.
+   Your password must contain letters and numbers
+
+#.  Run Kitsune::
+
+       ./manage.py runserver 0.0.0.0:8000
+
+Now, just navigate to <http://localhost:8000> to see the application!
+
+
+.. _Errors:
+
+Errors during `vagrant up`
+--------------------------
+
+``vagrant up`` starts the virtual machine. The first time you run ``vagrant up`` it
+also `provisions <https://docs.vagrantup.com/v2/cli/provision.html>`_ the vm -
+i.e., it automatically installs and configures Kitsune software on the vm. We
+provision the vm with `vagrant_provision in the `/bin directory
+<https://github.com/mozilla/kuma/tree/master/bin>`_.
+
+Sometimes we put vagrant declarations in the wrong order. Which means some
+errors can be fixed by simply provisioning the vm again::
+
+       vagrant provision
+
+In some rare occasions you might need to run this multiple times. If you find an
+error that is fixed by running ``vagrant provision`` again, please email us the
+error at dev-sumo@lists.mozilla.org and we'll see if we can fix it.
+
+If you see the same error over and over, please ask for help in our mailing list.
+
+kitsune_south_migrate
+~~~~~~~~~~~~~~~~~~
+
+If you see errors like::
+
+    notice: /Stage[main]/Kitsune_config/Exec[kitsune_south_migrate]
+
+try to manually run database migrations in the vm. To do so::
+
+    vagrant ssh
+    python manage.py migrate
+
+If you get an error, please ask for help by email.
+
+Ubuntu
+~~~~~~
+
+On Ubuntu, ``vagrant up`` might fail after being unable to mount NFS shared
+folders. First, make sure you have the nfs-common and nfs-server packages
+installed and also note that you can't export anything via NFS inside an
+encrypted volume or home dir.
```

Comment:
I'm pretty sure this part about encrypted drives and encrypted homedirs is just wrong, considering I use an encrypted drive, and have used Vagrant. Do you have any evidence to back up this claim?